go.mod

```diff
@@ -17,6 +17,7 @@ require (
 	github.com/gen2brain/beeep v0.0.0-20180718162406-4e430518395f
 	github.com/getkin/kin-openapi v0.22.0
 	github.com/go-chi/chi v4.1.2+incompatible // indirect
+	github.com/go-swagger/go-swagger v0.25.0
```

Comment:
Do we want this dependency in `go.mod` since it is used by our build process but not depended on by our software?

Reply:
It's currently required by our makefile.. so I would lean toward "yes".
But that's not the only way to achieve that : 
we have have another go.mod/go.sum in a separate directory, and fetch the go-swagger from there.

I think that it makes it simpler to know that all the dependencies are in one location, though.